demos/inputs.html

```diff
@@ -0,0 +1,295 @@
+<!doctype html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>CSS Chassis - Inputs</title>
+		<meta name="description" content="Button skeleton for styling">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="../dist/css/chassis.css">
+		<link rel="stylesheet" href="demos.css">
+	</head>
+	<body>
+
+		<h2>Inputs - (Size: Default)</h2>
+
+		<h3>Checkboxes</h3>
+		<!-- Custom check boxes -->
+
+		<input id="check1" type="checkbox" class="checkbox">
+		<label for="check1" class="checkbox-label">Choose this</label>
+
+		<input id="check2" type="checkbox" class="checkbox">
+		<label for="check2" class="checkbox-label">And this</label>
+
+		<input id="check3" type="checkbox" class="checkbox">
+		<label for="check3" class="checkbox-label">And this too</label>
+
+		<!-- Custom Radio button -->
+
+		<h3>Radio buttons</h3>
+
+		<div>
+			<input id="radio-1" class="radiobutton" name="radio-group" type="radio" checked>
```

Comment:
All of these values are checked, but only one radio element can be checked. That's why the demo is defaulting to the last radio button.